src/AlertBanner/AlertBanner.story.mdx

```diff
@@ -0,0 +1,91 @@
+import { AlertBanner } from "../AlertBanner";
+import {
+  Description,
+  Meta,
+  Story,
+  Props,
+  Preview,
+} from "@storybook/addon-docs/blocks";
+
+<Meta title="Components|AlertBanner" />
+
+# AlertBanner
+
+<Description
+  of={AlertBanner}
+  markdown="**AlertBanners** are primarily for situations where there is nothing you can do in the UI to immediately fix or change the state (e.g. deprecation warnings, account status alerts, etc)."
+/>
+
+## Info Alert
+
+<Preview>
+  <Story name="info light">
+    <AlertBanner type="info">
+      This is an example of an inline info message. It should be rather short,
+      but can have multi-lines if needed.
+    </AlertBanner>
+  </Story>
+  <Story name="info dark">
+    <AlertBanner type="info" theme="dark">
+      This is an example of an inline info message. It should be rather short,
+      but can have multi-lines if needed.
+    </AlertBanner>
+  </Story>
+</Preview>
+
+## Warning Alert
+
+<Preview>
+  <Story name="warning light">
+    <AlertBanner type="warn">
+      This is an example of an inline warning message. It should be rather
+      short, but can have multi-lines if needed.
+    </AlertBanner>
+  </Story>
+  <Story name="warning dark">
+    <AlertBanner type="warn" theme="dark">
+      This is an example of an inline warning message. It should be rather
+      short, but can have multi-lines if needed.
+    </AlertBanner>
+  </Story>
+</Preview>
+
+## Error Alert
+
+<Preview>
+  <Story name="error light">
+    <AlertBanner type="error" style={{ marginBottom: 20 }}>
+      This is an example of an inline error message. It should be rather short,
+      but can have multi-lines if needed.
+    </AlertBanner>
+  </Story>
+  <Story name="error dark">
+    <AlertBanner type="error" theme="dark" style={{ marginBottom: 20 }}>
+      This is an example of an inline error message. It should be rather short,
+      but can have multi-lines if needed.
+    </AlertBanner>
+  </Story>
+</Preview>
+
+## Success Alert
+
+<Preview>
+  <Story name="success light">
+    <AlertBanner type="success">
+      This is an example of an inline error message. It should be rather short,
+      but can have multi-lines if needed. Learn more
+    </AlertBanner>
+  </Story>
+  <Story name="success dark">
+    <AlertBanner type="success" theme="dark">
+      This is an example of an inline error message. It should be rather short,
+      but can have multi-lines if needed. Learn more
```

Comment:
The `Learn more` in these should really be linked up (or) to demonstrate how to use links within the banner. Or if you were thinking that the whole banner would be a target, that works too, but the underline on the `Learn more` text is still a helpful visual clue.

Reply:
oops, I meant to remove the learn nows to simplify the story :D
I'll make another PR to clean this up and remove the shadow 👍